Customer_Application/Customer_Api/Infrastructure/DataBaseOperation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Customer_Api.Model;
+
+namespace Customer_Api.Infrastructure
+{
+    public interface IDataBaseOperation
+    {
+        public string GetAllCustomerData();
+        public string getbyid(int id);
+        public string getbyname(string customername);
+        public string addcustomer(CustomerData newcustomer);
+        public string updatecustomer(CustomerData newcustomer);
+        public string deletecustomer(int id);
+         
+    }
+
+
```

Comment:
Create new lib project as infrastructure and move it

Reply:
database opearation added in class library project Infrastructure